packages/inter-protocol/test/psm/gov-add-psm-permit.json

```diff
@@ -0,0 +1,52 @@
+{
+  "consume": {
+    "agoricNamesAdmin": true,
+    "bankManager": true,
+    "board": true,
+    "chainStorage": true,
+    "zoe": "zoe",
+    "feeMintAccess": "zoe",
+    "economicCommitteeCreatorFacet": "economicCommittee",
+    "chainTimerService": "timer"
```

Comment:
why does it need the timer service?  governance?

Reply:
yes.

in `startPSM.js`:

```js
  const governorTerms = await deeplyFulfilledObject(
    harden({
      timer: chainTimerService,
      economicCommittee,
...
  );
```